File: Python/PRAK301-2210817210015-MuhammadIrfan.py
a = int(input("Masukkan bilangan pertama yang ingin diurutkan: "))
b = int(input("Masukkan bilangan keduayang ingin diurutkan   : "))
c = int(input("Masukkan bilangan ketiga yang ingin diurutkan : "))
if a>b and a>c:
    if b>c:
        print(c, b, a)
    else:
        print(b, c, a)
elif b>a and b>c:
    if a>c:
        print(c, a, b)
    else:
        print(a, c, b)
else:
    if a>b:
        print(b, a, c)
    else:
        print(a, b, c)

File: C/PRAK301-2210817210015-MuhammadIrfan.c
#include<stdio.h>
    int main(){
        int a, b, c;
        printf("Masukkan bilangan pertama yang ingin diurutkan: "); scanf("%d", &a);
        printf("Masukkan bilangan kedua yang ingin diurutkan  : "); scanf("%d", &b);
        printf("Masukkan bilangan ketiga yang ingin diurutkan : "); scanf("%d", &c);
        if(a>b && a>c && b>c){
            printf("%d, %d, %d", c, b, a);}
            else if (a>b && a>c && b<c){
                printf("%d, %d, %d", b, c, a);}
            else if (b>a && b>c && a>c){
                printf("%d, %d, %d", c, a, b);}
            else if (b>a && b>c && a<c){
                printf("%d, %d, %d", a, c, b);}
            else if (c>a && c>b && a>b){
                printf("%d, %d, %d", b, a, c);}
            else{
                printf("%d, %d, %d", a, b, c);}
        return 0;
    }